src/main.rs

```diff
@@ -59,24 +59,24 @@ lazy_static! {
 // ========================================================
 // Not-so glorious main function
 // ========================================================
-fn main() {
+fn main() -> Result<(), Box<dyn std::error::Error>> {
 	std::panic::set_hook(Box::new(handle_panic::handle_panic));
 
-	let img = image::load_from_memory(include_bytes!("../resource/emulsion48.png")).unwrap();
+	let img = image::load_from_memory(include_bytes!("../resource/emulsion48.png"))?;
```

Comment:
I don't see the advantage of returning with an error from `main` opposed to panicing on the error. Furthermore if this specific function wouldn't be inside the main function, panicing would still be the most sensible thing to do here. This is an image file that's embbeded into the emulsion executable and one that we tested to make sure it can be loaded. This function simply must not fail. If it does that's a situation where there's no point in trying to recover.